### PR TITLE
Migrate mysql domain service to new module system

### PR DESCRIPTION
**Pull request by DanielFran**
Fix https://github.com/jhipster/jhipster-lite/issues/2113

### CODE REVIEW

File: src/test/java/tech/jhipster/lite/generator/history/infrastructure/primary/GeneratorHistoryInterceptorIT.java [hunk 1]
```diff
@@ -28,7 +28,7 @@ void shouldNotAddHistoryOnUnknownFolder() throws Exception {
 
     mockMvc
       .perform(
-        post("/api/servers/spring-boot/databases/mysql")
+        post("/api/servers/spring-boot/databases/mssql")
```
Comment (pascalgrimaud) [inline src/test/java/tech/jhipster/lite/generator/history/infrastructure/primary/GeneratorHistoryInterceptorIT.java:31]: Just wonder why this change? Does it work only with MSSQL now?

Comment (DanielFran) [reply src/test/java/tech/jhipster/lite/generator/history/infrastructure/primary/GeneratorHistoryInterceptorIT.java:31]: @pascalgrimaud need to check, with modules, when calling an endpoint that is not available, the rest API returns not found instead of the expected bad request

But it is something to review in another issue...
